docs/notebooks/MetaOT.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<p>For all citations, please try to use the references already present in <a href="https://github.com/ott-jax/ott/blob/main/docs/references.bib" target="_blank">references.bib</a>. They can be added as, e.g., <code>&lt;cite data-cite="amos:22"&gt;&lt;/cite&gt;</code>.</p><p>Missing ending <code>.</code> , the 1st/3rd item in the list (or extra <code>.</code> at the end of the 2nd list item, w/e you prefer ).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Do you know if there's a way to do inline citations with \citet? I can't easily find anything in these docs (not sure if they're the right ones):</p><p> </p><p>https://nbconvert.readthedocs.io/en/latest/api/filters.html#nbconvert.filters.citation2latex</p><p>https://readthedocs.org/projects/sphinxcontrib-bibtex/downloads/pdf/latest/</p>

Reply:
<p>For now, I updated the text so the \citep makes sense</p>

Reply:
Thank you! 

> Do you know if there's a way to do inline citations with \citet? I can't easily find anything in these docs (not sure if they're the right ones):

I'm afraid this is not possible to do in the notebooks, the parser always outputs `\cite{}`, see [here](https://github.com/jupyter/nbconvert/blob/main/nbconvert/filters/citation.py#L46), but seems like an easy extension.